src/components/typography/Text.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import typography from "@/themes/typography";
+import styled from "styled-components";
+
+const { text } = typography;
+
+const Text = styled.div(({ className, paragraph, link }) => [
+  text.default.base,
+  text.default.color,
+  text.default.size,
+  link && text.link.base, // CAN YOU PLEASE MAKE
+  link && text.link.color, // THESE THREE LINES
+  link && text.link.size, // DRYer?  THANKS.
+  paragraph && text.paragraph.base, // CAN YOU PLEASE MAKE
+  paragraph && text.paragraph.color, // THESE THREE LINES
+  paragraph && text.paragraph.size, // DRYer?  THANKS.
```

Comment:
Please note the comment here about making these DRYer (if possible).

Reply:
It seems impossible to me

Reply:
@MaksymTaratuta55 

Okay -- so let's leave it as it is.  Can you please remove the comments?